journal/src/test/java/io/camunda/zeebe/journal/file/JournalTest.java

```diff
@@ -191,7 +191,7 @@ void shouldReset() {
   }
 
   @Test
-  void shouldThrowExceptionWhenResetWhileReading() {
+  void shouldNotReadAfterResetWhileReading() {
```

Comment:
:upside_down_face: I get it, but it looks funky. I guess we want to test that you cannot read more after a journal reset _until_ you've reset your reader? But I don't know how to succintly put this :smile: `shouldNotReadAfterJournalResetWithoutReaderReset()`? :smile: